stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/TypeEvent.kt

```diff
@@ -0,0 +1,3 @@
+package io.getstream.chat.android.client.models
+
+public data class TypeEvent(val channelId: String, val users: List<User>)
```

Comment:
One last request: we call this _typing_ everywhere, including all the LiveDatas and also the events that eventually map into this (`TypingStartEvent`, `TypingStopEvent`). Let's call this `TypingEvent` as well.

(This current name also feels a bit like it's an event about the type of something.)